src/config/settings.ts

```diff
@@ -56,7 +56,7 @@ export const defaultSettings = {
   /** Whether we should check for updates and apply them. */
   updater:                true,
   debug:                  false,
-  pathManagementStrategy: PathManagementStrategy.Manual,
+  pathManagementStrategy: PathManagementStrategy.RcFiles,
```

Comment:
Opted to default to `RcFiles` because the `Manual` configuration can leave a user with a broken installation if they don't take any action.

Reply:
That's a good catch. My only concern with invoking `.enforce()` is that we might be modifying PATH before a selection is made.. do we have the concept of No Strategy for the path manager, as in a strategy needs to be chosen? 

An alternative approach might be to check if this is the first run or an upgrade and make sure that the path manager is always enforced in these scenarios.

Reply:
In practice `ManualPathManager` is no strategy - it does nothing. But I guess conceptually it isn't no strategy. It would be easy to allow `pathManager` to have a value of `undefined`. Or we could introduce a `PathManager` implementation that does the same thing as `ManualPathManager` but is called `NoStrategyPathManager`.

But it seems like we should be able to ensure that the user takes action on the first run/upgrade dialog. Then we don't have to worry about this case at all. Is this possible?